scripts/install

```diff
@@ -447,7 +449,7 @@ cp ../conf/dnsmasq_regenconf_hook /usr/share/yunohost/hooks/conf_regen/50-dnsmas
 # ADVERTISE SERVICE IN ADMIN PANEL
 #=================================================
 
-yunohost service add pihole-FTL --description "PiHole backend service" --log "/var/log/pihole-FTL.log"
+yunohost service add pihole-FTL --description="PiHole backend service" --log="/var/log/pihole-FTL.log" --needs_exposed_ports="$port"
```

Comment:
https://docs.pi-hole.net/main/prerequisites/

> FTL is our API engine and uses port 4711 on the localhost interface. This port should not be accessible from any other interface.

We absolutely don't want `needs_exposed_ports` here